extensions/amp-experiment/1.0/test/test-attribute-mutation.js

```diff
@@ -75,71 +76,105 @@ describes.realWin(
     }
 
     function getAttributeMutationDefaultUrl(value) {
-      const paramsObject = getAttributeMutationParamsObject('href', value, 'a');
+      elements = [doc.createElement('a')];
+      const paramsObject = getAttributeMutationParamsObject('href', value);
       return new AttributeMutationDefaultUrl(
         paramsObject.mutationRecord,
         paramsObject.elements
       );
     }
 
-    describe('validate', () => {
+    describe('parseAndValidate', () => {
       describe('default class', () => {
         it('should allow valid mutations', () => {
           const attributeMutation = getAttributeMutationDefaultClass(
             'my-class'
           );
-          expect(attributeMutation.validate()).to.be.equal(true);
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
```

Comment:
nit:
to.equal(true)

or

to.be.true

Reply:
hmmm, that require me to change this everywhere. You want me to change it in this PR?

Reply:
done